modules/025-static-routing-manager/images/agent/pkg/logger/logger.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2024 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logger
+
+import (
+	"flag"
+	"fmt"
+	"github.com/go-logr/logr"
+	"k8s.io/klog/v2"
+	"k8s.io/klog/v2/klogr"
+)
+
+const (
+	ErrorLevel   Verbosity = "0"
+	WarningLevel Verbosity = "1"
+	InfoLevel    Verbosity = "2"
+	DebugLevel   Verbosity = "3"
+	TraceLevel   Verbosity = "4"
+)
+
+const (
+	warnLvl = iota + 1
+	infoLvl
+	debugLvl
+	traceLvl
+)
+
+type (
+	Verbosity string
+)
+
+type Logger struct {
+	log logr.Logger
+}
+
+func NewLogger(level Verbosity) (*Logger, error) {
```

Comment:
NIT: The move with logger is puzzling. There is stdlib `log/slog`. Cann't we just use it, or stick to the klog?

Reply:
It is better to not blindly snatch things. With the current approach, this code will stay in the codebase like forever. Nobody will refactor it.

It is better to do it now.

Reply:
fixed